symplyphysics/laws/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits.py

```diff
@@ -0,0 +1,56 @@
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Kepler's third law is as follows. The squares of the periods of rotation of the planets around the Sun are referred to
+## as cubes of the large semi-axes of the orbits of the planets. But Newton found that Kepler's third law is not entirely
+## accurate — in fact, it includes the masses of the planets and the Sun.
+
+## Law is: ((M + m1) * T1^2) / ((M + m2) * T2^2) = a1^2 / a2^2, where
```

Comment:
Does not correspond to description.

Reply:
Not fixed.

Reply:
> Not fixed.

I don't understand what the discrepancy is with the description.